calculations/calculations.py

```diff
@@ -61,50 +59,66 @@ def estimate_damping_force(self) -> pd.DataFrame:
-
-    def calculate_forces_part_3_rear(self) -> float:
-        return self.K_R * (self.data["Front Left_lowpass"] - self.data["Front Right_lowpass"] +
-                           self.data["Rear Left_lowpass"] - self.data["Rear Right_lowpass"]) * (1 - self.q)
-
-    def calculate_forces_part_4_front(self) -> float:
-        return self.K_W * (self.data["Front Left_lowpass"] - self.data["Front Right_lowpass"] -
-                           self.data["Rear Left_lowpass"] + self.data["Rear Right_lowpass"]) * (1 - self.w)
-
-    def calculate_forces_part_4_rear(self) -> float:
-        return self.K_W * (self.data["Front Left_lowpass"] - self.data["Front Right_lowpass"] -
-                           self.data["Rear Left_lowpass"] + self.data["Rear Right_lowpass"]) * self.w
-
-    def calculate_force_front_left(self) -> float:
-        self.data["Force Front Left"] = constants.CORNER_WEIGHT_LF + 0.25 * (self.calculate_forces_part_1(
-        ) + self.calculate_forces_part_2() + self.calculate_forces_part_3_front() + self.calculate_forces_part_4_front())
-        return self.data["Force Front Left"]
-
-    def calculate_force_front_right(self) -> float:
-        self.data["Force Front Right"] = constants.CORNER_WEIGHT_RF + 0.25 * (self.calculate_forces_part_1(
-        ) + self.calculate_forces_part_2() - self.calculate_forces_part_3_front() - self.calculate_forces_part_4_front())
-        return self.data["Force Front Right"]
-
-    def calculate_force_rear_right(self) -> float:
-        self.data["Force Rear Right"] = constants.CORNER_WEIGHT_RR + 0.25 * (self.calculate_forces_part_1(
-        ) - self.calculate_forces_part_2() + self.calculate_forces_part_3_rear() - self.calculate_forces_part_4_rear())
-        return self.data["Force Rear Right"]
-
-    def calculate_force_rear_left(self) -> float:
-        self.data["Force Rear Left"] = constants.CORNER_WEIGHT_RL + 0.25 * (self.calculate_forces_part_1(
-        ) - self.calculate_forces_part_2() - self.calculate_forces_part_3_rear() - self.calculate_forces_part_4_rear())
-        return self.data["Force Rear Left"]
+    
+    def calculate_wheel_loads(self, df: pd.DataFrame) -> pd.DataFrame:
+        df_after_front_left = self.calculate_force_front_left(df)
+        df_after_front_right = self.calculate_force_front_right(df_after_front_left)
+        df_after_rear_left = self.calculate_force_rear_left(df_after_front_right)
+        return self.calculate_force_rear_right(df_after_rear_left)
+
+    def calculate_wheel_load_constants(self):
+        self.calculate_wheel_rate_front()
+        self.calculate_wheel_rate_rear()
+        self.calculate_wheel_rate_roll_front()
+        self.calculate_wheel_rate_roll_rear()
+        self.calculate_K_H_K_P()
+        self.calculate_K_R_K_W()
+        self.calculate_a_b()
+        self.calculate_q_w()
+
+    def calculate_forces_part_1(self, df: pd.DataFrame) -> float:
+        return self.K_H * (df["Front Left_lowpass"] + df["Front Right_lowpass"] +
```

Comment:
would you be able to either write comments about what K_H, K__P and some of these other variables are or have more meaningful names for them

Reply:
I think we can leave this is as alow priority item for after comp.

Reply:
its needed to place the wheel load methods into functions, im having a hard time following what it is you're doing so i can transfer this over, this is needed before comp

Reply:
Oh I see.